data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,3447 @@
+
+
+
+mission "Unfettered: Bring Medical Supplies Notification"
+	landing
+	name `Need for medical supplies`
+	description `An Unfettered elder told you to come help the hospitals on <destination> that desperately need supplies.`
+	source "Firelode"
+	destination "Darkcloak"
+	to offer
+		has "Unfettered: Bring Hai: done"
+
+
+
+mission "Unfettered: Bring Medical Supplies"
+	landing
+	name `Bring medicals to <planet>`
+	description `The Unfettered hospitals on Darkcloak are in urgent need of resupply. Collect <tons> of medical supplies from <stopovers>, and bring it to the Unfettered on <destination> by <date> for <payment>.`
+	source "Darkcloak"
+	stopover "Snowfeather"
+	cargo "medicals" 120
+	blocked `You need 120 cargo space available to carry the medicals supplies.`
+	deadline 8
+	to offer
+		has "Unfettered: Bring Hai: done"
+	on offer
+		conversation
+			`As you land, you receive a distress message, originating from one of the hospitals nearby. Staff are asking all who can to immediately bring back medical supplies from <stopovers>.`
+			`	When landing on an Unfettered planet, even after you brought them a jump drive, there is always some lingering tension. Now you feel the tension much more than usual, probably because of the medical emergency. Unfettered are all around, launching small ships to bring the wounded to the hospitals as quickly as possible. The hospital itself looks different than the rest of the Unfettered structures, it seems more organized, proper. It is probably the best maintained structure, or at least has that appearance, because it is the most useful to them, alongside the shipyards. But one part of it looks to have been grafted onto the rest, and not by a well trained physician, to say the least... You figure they must really be needing more space for their sick.`
+			`	Considering all of this, the Unfettered may not take no for an answer... Plenty of ships are already taking off to try and get those much needed medicals.`
+			choice
+				`	(Signal them I am going to get the supplies.)`
+					accept
+				`	(I do not have time for this.)`
+			`	Thankfully it seems that everyone is too busy with the local problem to annoy you or press you into accepting this mission, but this will probably have consequences later. You better not stay landed for too long, or you could draw attention.`
+	
+	on visit
+		dialog phrase "generic cargo on visit"
+	on stopover
+		dialog `You are surprised to see the Hai are more than happy to give you the medical supplies needed. They load up the cargo and once they are done one of they addresses you, "Tell our brothers and sisters to stay strong, hopefully it is not too late," he says with determination. Hopefully these supplies will help to make a difference.`
```

Comment:
`Hopefully these supplies will help to make a difference.`
What is this line? Is this spoken, or is this something you think?
If spoken it needs to be spoken, otherwise it probably needs rework as it feels very out of place to describe the player thinking this.

Reply:
Unless it's adjusted to be spoken by a character, yes.